src/api/app/models/bs_request.rb

```diff
@@ -755,7 +755,7 @@ def notify_parameters(ret = {})
     ret[:description] = self.description
     ret[:state] = self.state
     ret[:oldstate] = self.state_was if self.state_changed?
-    ret[:who] = User.current.login
+    ret[:who] = self.commenter unless self.commenter.blank?
```

Comment:
since it can be transfered from backend I think it is safer to ignore also empty strings